web-platform/website/src/components/pages/account-settings/AccountSettingsPageContent.tsx

```diff
@@ -59,6 +85,16 @@ const styles = {
       ".nav": {
         marginRight: Spacing.xxxLarge, // Aligns with top navigation logo column
       },
+      ".account-toast": {
+        backgroundColor: Colors.ColorGreen10,
+        width: "302px",
+        fontSize: "0.938rem",
```

Comment:
Can we use standard values here from FontSizes / Spacing? I have tried very hard not to use values that are non-standard.

Reply:
Yes, trying to align with what Casey has in his designs.